### ISSUE TITLE
Issue with IE10 recognizing enquire

### ISSUE THREAD

**Issue by nickff** [WickyNilliams/enquire.js#88]
Hey Nicky,

So I have a site that I've been using enquire on for about a year with no issues (http://jamesmerrellarchitects.com). I was just testing the site on IE10, and unfortunately I'm getting an error (works great in IE9 though).

The error I'm getting is:
SCRIPT5009: 'enquire' is undefined 
scripts.js, line 263 character 5

I'm running enquire within this file, right towards the bottom - http://jamesmerrellarchitects.com/assets/js/scripts.js

And here's my code:

```
//animate the icons differently depending on small screen or not.
Modernizr.load([
    {
        test: window.matchMedia,
        nope: "assets/js/matchMedia.js"
    },
    {
        load : "assets/js/enquire.js",
        complete : function() {
            $(function() {
                enquire.register("screen and (min-width:581px)", { 
                    match : function() {
                        INIT.menu_animate();
                    }
                }, true).register("screen and (max-width:580px)", {
                    match : function() {
                        INIT.mobile_menu_animate();
                    }
                }).fire();  
            });
        }
    }
]);
```

Basically, it animates the bottom menu icons differently based on screensize.

If you have any thoughts on why this would be giving me an error in IE10, it would be hugely appreciated!
